internal/server/cookie_test.go

```diff
@@ -44,12 +45,13 @@ func TestResendAuthCookie(t *testing.T) {
 	}
 	c.Set(access.RequestContextKey, rCtx)
 
-	exchangeSignupCookieForSession(c, baseDomain)
+	exchangeSignupCookieForSession(c, Options{SessionDuration: 1 * time.Minute, BaseDomain: baseDomain})
 
 	assert.Equal(t, len(c.Writer.Header()["Set-Cookie"]), 2)
-	expectedCookies := []string{
-		"auth=aaa; Path=/; Domain=dev.example.com; Max-Age=299; HttpOnly; SameSite=Strict",
-		"auth=; Path=/; Domain=example.com; Max-Age=0; HttpOnly; Secure",
-	}
-	assert.DeepEqual(t, c.Writer.Header()["Set-Cookie"], expectedCookies)
+
+	matched, err := regexp.MatchString("auth=aaa; Path=/; Domain=dev.example.com; Max-Age=\\d\\d; HttpOnly; SameSite=Strict", c.Writer.Header()["Set-Cookie"][0])
+	assert.NilError(t, err)
+	assert.Assert(t, matched)
```

Comment:
One option to avoid the regex would be to use store the `httptest.NewRecorder()` that we passed into the `gin.Context` and call `Result().Cookies()` on it (https://pkg.go.dev/net/http#Response.Cookies). 

That should give you the parsed structs, and allow you to use `DeepEqual` with options to account for unpredictable max age.

Not a blocker, just an option if we need to make further changes some other time.